packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -44,7 +47,17 @@ export function Menu({
   useEffect(() => setHighlightedIndex(initialHighlight), [options]);
 
   return (
-    <div className={optionMenuClass}>
+    <div
+      className={optionMenuClass}
+      style={{ scrollBehavior: "smooth" }}
+      ref={ref => {
+        if (!menuDiv.current) {
+          const stuff = ref as HTMLDivElement;
+          menuDiv.current = stuff;
+        }
+        return menuDiv.current;
+      }}
```

Comment:
I don't understand why you need to build the ref this way? Can'y you just set `ref={menuDiv}`?

Reply:
@willcodeforcoffee It works, but typescript doesn't seem to like that

![Screen Shot 2020-09-14 at 1 46 07 PM](https://user-images.githubusercontent.com/34727471/93136392-b32cc280-f690-11ea-9870-963194ab0831.png)